ui/shared/components/panel/variants/selectors.js

```diff
@@ -135,9 +136,19 @@ export const getPairedSelectedSavedVariants = createSelector(
     } else if (tag === MME_TAG_NAME) {
       pairedVariants = matchingVariants(pairedVariants, ({ mmeSubmissions = [] }) => mmeSubmissions.length)
     } else if (tag && tag !== SHOW_ALL) {
-      pairedVariants = matchingVariants(
-        pairedVariants, ({ tagGuids }) => tagGuids.some(tagGuid => tagsByGuid[tagGuid].name === tag),
-      )
+      const tags = tag.split(TAG_URL_DELIMITER)
```

Comment:
Doesn't doing the split here means all the above logic for MME/notes would not work?

Reply:
`MME Submission/Has Notes` won't work for multiple-tag filtering. They are not required since they are not on the summary data options list. But we should have a more systemic solution for handling tag/tags here.

Reply:
Since we filter the tags in the backend and don't support the project saved-variant multiple-tag filtering, these changes are unnecessary.